src/domUpdates.js

```diff
@@ -0,0 +1,32 @@
+export const domUpdates = {
```

Comment:
I like this as a single object. Definitely cleans it up nicely... maybe I should try this.